sartography/spiff-arena
# Feature/business end states

This will display last milestone bpmn name as last milestone in the process instances table and on the process instance show page. Start and End events are ignored unless they have explicit names.

To test:
Create a process model with an intermediate throw event, start event, or end event with a name and ensure after completing that task, the last milestone on the process instance show page displays the name of that event. If the instance is complete and the final end event that was completed does not have a name, then the last milestone will be "Completed".
